doc/release-notes.rst

```diff
@@ -171,6 +200,19 @@ Major Changes from Kraken
     - Ubuntu 16.04 Xenial (x86_64 and aarch64)
     - Ubuntu 14.04 Trusty (x86_64)
 
+  * A first release of Ceph for FreeBSD is available which contains a full set
+    of features, other than Bluestore. It will run everything needed to build a
+    storage cluster. For clients, all access methods are available, albeit
+    CephFS is only accessible through a Fuse implementation. RBD images can be
+    mounted on FreeBSD systems through rbd-ggate
+
+    Ceph versions are released through the regular FreeBSD ports and packages
+    system. The most current version is available as: net/ceph-devel. Once
+    Luminous goes into official release, this version will be available as
+    net/ceph. Future development releases will be available via net/ceph-devel
+
+    More details about this port are in: `README.FreeBSD <https://github.com/ceph/ceph/blob/master/README.FreeBSD>`_
```

Comment:
Pretty sure the multiple lines in this section are causing the blank-line-after-top-leave-headings render ugliness.

Reply:
Yeah I'll fix these